frontend/src/scenes/dashboard/TileFiltersOverride.tsx

```diff
@@ -0,0 +1,127 @@
+// scenes/dashboard/TileFiltersOverride.tsx
+import { BindLogic, useActions, useValues } from 'kea'
+
+import { IconCalendar } from '@posthog/icons'
+import { LemonButton, LemonDivider } from '@posthog/lemon-ui'
+
+import { DateFilter } from 'lib/components/DateFilter/DateFilter'
+import { PropertyFilters } from 'lib/components/PropertyFilters/PropertyFilters'
+import { TaxonomicFilterGroupType } from 'lib/components/TaxonomicFilter/types'
+import { TaxonomicBreakdownFilter } from 'scenes/insights/filters/BreakdownFilter/TaxonomicBreakdownFilter'
+import { insightLogic } from 'scenes/insights/insightLogic'
+
+import { groupsModel } from '~/models/groupsModel'
+import { BreakdownFilter, NodeKind } from '~/queries/schema/schema-general'
+import type { DashboardTile, InsightLogicProps, QueryBasedInsightModel } from '~/types'
+
+import { tileLogic } from './tileLogic'
+
+export function TileFiltersOverride({
+    tile,
+    dashboardId,
+}: {
+    tile: DashboardTile<QueryBasedInsightModel>
+    dashboardId: number
+}): JSX.Element {
+    const { overrides, hasOverrides } = useValues(tileLogic)
+    const { setDates, setProperties, setBreakdown, resetOverrides } = useActions(tileLogic)
+    const { groupsTaxonomicTypes } = useValues(groupsModel)
+    const insightProps: InsightLogicProps = {
+        dashboardItemId: tile.insight?.short_id,
+        dashboardId: dashboardId,
+        cachedInsight: null,
+        query: {
+            kind: NodeKind.InsightVizNode,
+            source: {
+                kind: NodeKind.TrendsQuery,
+                series: [],
+            },
+        },
+    }
+
+    return (
+        <div className="space-y-4">
+            <div>
+                <p className="text-sm text-muted mb-4">
+                    Set custom filters for this tile that will override the dashboard's global filters.
+                </p>
+            </div>
+
+            <div className="space-y-4">
+                <div>
+                    <label className="text-sm font-medium mb-2 block">Date Range</label>
+                    <DateFilter
+                        showCustom
+                        dateFrom={overrides.date_from ?? null}
+                        dateTo={overrides.date_to ?? null}
+                        onChange={(from, to) => setDates(from, to)}
+                        makeLabel={(key) => (
+                            <>
+                                <IconCalendar />
+                                <span className="hide-when-small"> {key}</span>
+                            </>
+                        )}
+                    />
+                </div>
+
+                <div>
+                    <label className="text-sm font-medium mb-2 block">Properties</label>
+                    <PropertyFilters
```

Comment:
The add filter button looks off being so wide
<img width="540" height="303" alt="Image" src="https://github.com/user-attachments/assets/1c148e6e-d594-4af0-8f98-b5d6f71ed225" />